analyses/snv-callers/scripts/01-calculate_vaf_tmb.R

```diff
@@ -131,6 +141,15 @@ opt$bed_wgs <- file.path(root_dir, opt$bed_wgs)
 opt$bed_wxs <- file.path(root_dir, opt$bed_wxs)
 opt$cosmic <- file.path(root_dir, opt$cosmic)
 
+# Bring along the file suffix. Make to lower.
+file_suffix <- tolower(opt$file_format)
+
+# Check that the file format is supported
+if (!(file_suffix %in% c('rds', 'tsv'))) {
+  stop("Option used for file format (-f) is not supported. Only 'tsv' or 'rds'
```

Comment:
```suggestion
  warning("Option used for file format (-f) is not supported. Only 'tsv' or 'rds'
```
And then say "defaulting to RDS" ?

Reply:
Okay. Will do.